### ISSUE TITLE
Error 1061 (42000): Duplicate key name issues when using MySQL Aria storage engine

### ISSUE THREAD

**Issue by miqraeng** [go-gitea/gitea#31210]
### Description

When restoring the gitea database form a backup I got a lot of messages of type
Error 1061 (42000): Duplicate key name ....

I got those same messages when doing an upgrade, and when restoring the original database.
The command ```gitea doctor check-db-version``` gave these messages.

After some research I discovered that my MariaDB database engine was set to Aria
Recreating the database with InnoDB restored proper functionality

### Gitea Version

1.19.3 to 1.22.0

### Can you reproduce the bug on the Gitea demo site?

No

### Screenshots

![image](https://github.com/go-gitea/gitea/assets/142734208/fab3ea40-9904-441b-a403-04e06666c46a)

### Git Version

2.30.2

### Operating System

 Debian GNU/Linux 11 (bullseye) AMD64

### How are you running Gitea?

downloaded releases

### Database

MySQL/MariaDB

**Comment by lunny** [comment]
I think Gitea now just supports `innodb` Engine. https://docs.gitea.com/help/faq#upgrade-errors-with-mysql

**Comment by miqraeng** [comment]
True, but it would be nice if the '''gitea doctor ''' feature could identify the issue and possibly remedy it.
